### DISCUSSION TITLE
How to inner source the local send protocol in a Flutter app

### DISCUSSION THREAD

**Discussion by lcmohale** [localsend/localsend#2033] [Discussion (English, International)] [answered]
Hi all, I currently in the process of developing a Flutter community app, and I want to implement the app to be able to send files similar to local send, is there a doc, blog, on how to use the protocol? in own app?

**Comment by Tienisto** [comment, marked as answer]
The protocol is 50% implemented in the app directory and 50% implemented in the common directory. So it will be difficult. LocalSend is in the progress of being refactored so that the commons package is a standalone.
